logstash-filter-aggregate.gemspec

```diff
@@ -6,7 +6,7 @@ Gem::Specification.new do |s|
   s.description = "This gem is a logstash plugin required to be installed on top of the Logstash core pipeline using $LS_HOME/bin/plugin install gemname. This gem is not a stand-alone program"
   s.authors = ["Elastic", "Fabien Baligand"]
   s.email = 'info@elastic.co'
-  s.homepage = "http://www.elastic.co/guide/en/logstash/current/index.html"
+  s.homepage = "https://github.com/logstash-plugins/logstash-filter-aggregate"
```

Comment:
@purbon 
OK, I keep https://github.com/logstash-plugins/logstash-filter-aggregate as home page.
But I'm still strongly interested to add aggregate plugin here : https://www.elastic.co/guide/en/logstash/current/filter-plugins.html

Is it possible ? Does it require something particular to be published here ?